support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.ts

```diff
@@ -134,7 +134,7 @@ export type RegularPaymentRequest = {
 	firstName: string;
 	lastName: string;
 	billingAddress: RegularPaymentRequestAddress;
-	deliveryAddress?: RegularPaymentRequestAddress;
+	deliveryAddress?: RegularPaymentRequestAddress | null;
```

Comment:
Perhaps we should leave it undefined there and leave this definition as is? I would much prefer to standardise on 'value or undefined' rather than 'value or undefined or null'